src/views/canvas/CanvasPinnedBlock.js

```diff
@@ -52,9 +52,17 @@ const CanvasPinnedBlock = boneView.extend({
       this.ctx.fillRect(x + xOffset, y, width, height);
 
       // draw background block border
-      this.ctx.strokeStyle = 'black';
-      this.ctx.lineWidth = borderWidth;
+      this.ctx.strokeStyle = feature.attributes.verticalSeperatorColor || '#808080';
```

Comment:
Can you set `black` as the default?
Infact, I think the defaults can be filled in `Feature.js`

Reply:
Default values are set in Feature.js